### PR TITLE
Fix more links

### PR DESCRIPTION
**Pull request by chipzoller**
Signed-off-by: Chip Zoller <chipzoller@gmail.com>

Fixes more links exposed in the link checking action by direct fixes and absorbing changes in code-contribution.md into other sections.

### CODE REVIEW

**Comment by chipzoller** [comment]
This should resolve all but a couple broken links which require further discussion for how to best remediate.

File: README.md [hunk 1]
```diff
@@ -76,6 +76,11 @@ Additionally, check out our [blog ](https://blog.kubecost.com/blog/cost-monitori
 
 ## Staying in the loop
 
-You can stay up to date with Kubecost by following releases on [GitHub](https://github.com/kubecost/cost-analyzer-helm-chart/releases).
+You can stay up-to-date with Kubecost by following releases on [GitHub](https://github.com/kubecost/cost-analyzer-helm-chart/releases) and the official [Kubecost blog](https://blog.kubecost.com/).
+
+Follow us on social media:
```
Comment (bstuder99) [inline README.md:81]: We have a Contact Us page at the top of the docs site for this content

**Review by bstuder99 — CHANGES_REQUESTED** [review]

**Comment by bstuder99** [comment]
Beyond some minor tweaks, looks good

File: README.md (hunk 1 above)
Comment (chipzoller) [reply README.md:81]: Thanks for that and sorry I missed it!

**Comment by chipzoller** [comment]
cc @mattray, per conversation earlier, do you believe this language re the README adjustments is aligned to your preferred messaging? If not, can you suggest changes?

**Review by bstuder99 — APPROVED** [review]

**Pull request merged**